src/policy_metadata.rs

```diff
@@ -141,6 +142,51 @@ const fn _default_true() -> bool {
     true
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone, Validate)]
+#[serde(rename_all = "camelCase")]
+#[validate(schema(function = "validate_metadata_v2", skip_on_field_errors = false))]
+pub struct MetadataV2 {
```

Comment:
I'm lost, why do we need a Metadata `V2`?

Reply:
We don't need it indeed! resolving, irrelevant after the refactor.